weed/topology/store_replicate.go

```diff
@@ -87,7 +89,10 @@ func ReplicatedWrite(masterFn operation.GetMasterFn, grpcDialOption grpc.DialOpt
 					pairMap[needle.PairNamePrefix+k] = v
 				}
 			}
-
+			bytesBuffer := buffer_pool.SyncPool.Get().(*bytes.Buffer)
```

Comment:
Better simplify this line into one function call.

Reply:
@chrislusf Done